pipelines/shopify_dlt/__init__.py

```diff
@@ -0,0 +1,72 @@
+
+"""Fetches Shopify Orders and Products."""
+
+from typing import Any, Dict, Iterator, Iterator
+
+import shopify
+
+import dlt
+
+from dlt.extract.source import DltResource
+
+
+@dlt.source(name="shopify")
+def shopify_source(
+    private_app_password: str = dlt.secrets.value,
+    api_version: str = dlt.config.value,
```

Comment:
please use a default value for api version. default values can still be configured in `config.toml` but optionally

Reply:
done

Reply:
(not quite sure though if I did it right. intuitively the way I did it now it would not get the version from the config file, I did not find anything in the docs about default values of config items though)